### PR TITLE
649 login friendly messages

### PR DESCRIPTION
**Pull request by jxofficial**
PR Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

PR Checklist

- [x] - Latest `master` branch merged
- [x] - PR title descriptive (can be used in release notes)
- [x] - Passes Tests

## Description

Provide friendly messages for failed login. 
If email provided is invalid, error message "No account found, typo maybe?" is displayed.
If password provided is invalid, error message "Password does not match the user account" is displayed.

## Git Issues

_Closes #649_

### CODE REVIEW

**Comment by BenGamma** [comment]
Looks good to me !
I'm asking quick review from @chrismclarke  as he was the one following up the issue.

**Comment by jxofficial** [comment]
Thanks for the update! 

In the meantime, I have gone ahead and merged the latest master branch before the final review.

File: src/utils/messages.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// set of mappings to use for system->friendly messages
+// some are pulled from error messages, others are hardcoded into the platform but kept
+// here to make easier to change in the future
+const MESSAGES = {
```
Comment (catx23) [inline src/utils/messages.ts:4]: There is no reason to create this file since there is only one consumer :-)

**Review by chrismclarke — APPROVED** [review]
Sorry, didn't realise this one is waiting on me. I can see the changes are fairly straight-forward (adding the extra message and profile component bindings). I notice that you have also refactored the messages to a separate utils file, I'm happy that it isn't within the component although thinking about it I think it would make more sense to be in some form of standalone file to make it easier to change, add to or translate in the future (e.g `utils/strings.ts` or similar). But for now I'd say good to merge and we can worry about the rest later.

**Pull request merged**